aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,190 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+:::moniker range=">= aspnetcore-8.0"
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. This article shows the following for native AOT support in ASP.NET Core:
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
+
+## Getting started with .NET native AOT deployment in ASP.NET Core
+
+To help developers get started deploying with .NET native AOT in ASP.NET Core, use the:
+
+* AOT API template which includes customizations to remove unsupported components from the app.
+* ```dotnet new``` command to create a new ASP.NET Core API app that is configured to work with .NET native AOT:
+
+```powershell
+PS> dotnet new api -aot -n MyFirstAotWebApi && cd MyFirstAotWebApi
```

Comment:
Nit: I believe we typically tend to use `-o FolderName` in our samples.

Also, I notice this is a `powershell` text type. Do we want to include the tabbed panels for different targets or use the more generic CLI format so people don't assume this is Windows-only?